srcpkgs/nvidia/template

```diff
@@ -19,7 +19,7 @@ conflicts="xserver-abi-video>24_1"
 
 _pkg="NVIDIA-Linux-x86_64-${version}"
 distfiles="http://uk.download.nvidia.com/XFree86/Linux-x86_64/${version}/${_pkg}.run"
-checksum=649684fab4f37b178afb6cae012e7f9736377d89fa8abdfca26f24a13bbacf1f
+checksum=d50c77fc4fda2a5c5ab2af64524da8a3214077bd7daf0dbf7c1986e0ca05d711
 subpackages="nvidia-gtklibs nvidia-dkms nvidia-opencl nvidia-libs nvidia-libs-32bit"
```

Comment:
Btw, I'm doing some weird shit with this package. Do you think we could have a comment that `nvidia-gtklibs` subpackage has to go before `nvidia-libs`?

Reply:
Does subpackage ordering matter?

Reply:
Yes, because it first moves the gtk libs and then moves the rest of them. If `nvidia-libs` happens before `nvidia-gtklibs`, it fails because it can't find the gtklibs.